hoot-services/src/test/java/hoot/services/utils/DbUtilsTest.java

```diff
@@ -71,6 +74,48 @@ public void setAppContext() {
     @Test
     @Category(UnitTest.class)
     @Transactional
+    public void testDeleteTables() throws Exception {
+        long userId = MapUtils.insertUser();
+        long mapId = insertMap(userId);
+        System.out.println(mapId);
+        DbUtils.createQuery().getConnection().commit();
+        
+        assertTrue(DbUtils.mapExists(String.valueOf(mapId)));
+        assertTrue(checkForTables(mapId));
+        
+        MapUtils.deleteOSMRecord(mapId);
+        DbUtils.createQuery().getConnection().commit();
+        
+        try {
+           assertFalse(DbUtils.mapExists(String.valueOf(mapId)));
+        }catch(IllegalArgumentException e) {
+        	System.out.println("Expected error."); 
```

Comment:
Wow, mapExists is dumb.  Can you update it to catch the IllegalArgumentException thrown by getRecordIdForInputString and just set id to -1 in that case so mapExists returns false?